guard/src/commands/aws_meta_appender_tests.rs

```diff
@@ -47,7 +47,7 @@ fn append_cdk_metadata_test() -> Result<()> {
             unimplemented!()
         }
 
-        fn end_evaluation(&self, eval_type: EvaluationType, context: &str, msg: String, from: Option<PathAwareValue>, to: Option<PathAwareValue>, status: Option<Status>) {
+        fn end_evaluation(&self, eval_type: EvaluationType, context: &str, msg: String, from: Option<PathAwareValue>, to: Option<PathAwareValue>, status: Option<Status>, _cmp: Option<(CmpOperator, bool)>) {
             assert_ne!(msg.as_str(), "");
             assert_eq!(msg.starts_with("FIRST PART"), true);
```

Comment:
Can we declare "FIRST PART" as a constant and reuse in this test class?

Reply:
This is unrelated to this change. Can we postpone these as it will confuse the PR?